lightningd/peer_htlcs.c

```diff
@@ -735,6 +735,7 @@ static void htlc_accepted_hook_serialize(struct htlc_accepted_hook_payload *p,
 	json_object_start(s, "htlc");
 	json_add_amount_msat_only(s, "amount", hin->msat);
 	json_add_u64(s, "cltv_expiry", hin->cltv_expiry);
+	json_add_u64(s, "cltv_expiry_delta", hin->cltv_expiry - p->ld->topology->tip->height);
```

Comment:
Is not `cltv_expiry_delta` the diference between the `ctlv_expiry` of incoming minus the `cltv_expiry` of outgoing?

Reply:
Yeah, maybe call this "cltv_expiry_relative" and make it a signed int just in case?